components/Layout.js

```diff
@@ -0,0 +1,16 @@
+import Seo from "./SEO";
```

Comment:
I've created a layout wrapper that can be reused here so that pages always have the same features and it's impossible to miss (especially during changes)